LiveSplit.AutoSplitters.xml

```diff
@@ -4844,6 +4844,16 @@
         <Type>Component</Type>
         <Description>Auto Splitting is available. (By ShootMe)</Description>
     </AutoSplitter>
+    <AutoSplitter>
+        <Games>
+            <Game>KeyWe</Game>
+        </Games>
+        <URLs>
+            <URL>https://gist.githubusercontent.com/Alecat/084ba4995b5cfa86021d2ce841717ba5/raw</URL>
+        </URLs>
+        <Type>Script</Type>
+        <Description>Autosplitter that tallies IGT level times for full-game runs. Additional resources available at https://ale.chenonetta.com/keywe/</Description>
```

Comment:
The below is a possible change you could make for the checks to pass:
```suggestion
        <Description>Starts, resets automatically. Syncs to and sums up in-game time. Click 'Website' for additional resources.</Description>
        <Website>https://ale.chenonetta.com/keywe</Website>
```

Reply:
Ah great, thank you!